coderadar-rest/src/test/java/io/reflectoring/coderadar/rest/unit/useradministration/RefreshTokenControllerTest.java

```diff
@@ -12,19 +13,21 @@
 import org.springframework.http.HttpStatus;
 import org.springframework.http.ResponseEntity;
 
-class RefreshTokenControllerTest {
+class RefreshTokenControllerTest extends UnitTestTemplate {
 
-  private RefreshTokenUseCase refreshTokenUseCase = mock(RefreshTokenUseCase.class);
+  private final RefreshTokenUseCase refreshTokenUseCase = mock(RefreshTokenUseCase.class);
 
   @Test
   void refreshTokenSuccessfully() {
     RefreshTokenController testSubject = new RefreshTokenController(refreshTokenUseCase);
 
     RefreshTokenCommand command = new RefreshTokenCommand("accessToken", "refreshToken");
     Mockito.when(refreshTokenUseCase.refreshToken(command)).thenReturn("newAccessToken");
-    ResponseEntity<RefreshTokenResponse> responseEntity = testSubject.refreshToken(command);
+    ResponseEntity<Object> responseEntity = testSubject.refreshToken(command);
 
+    Assertions.assertTrue(responseEntity.getBody() instanceof RefreshTokenResponse);
+    RefreshTokenResponse responseBody = (RefreshTokenResponse) responseEntity.getBody();
     Assertions.assertEquals(HttpStatus.OK, responseEntity.getStatusCode());
-    Assertions.assertEquals("newAccessToken", responseEntity.getBody().getToken());
```

Comment:
`responseBody` wird nur ein einziges mal verwendet, daher sehe ich hier keinen Mehrwert darin, das in einer eigenen Variablen zu speichern.